bioblend/galaxy/jobs/__init__.py

```diff
@@ -70,6 +70,48 @@ def show_job(self, job_id, full_details=False):
 
         return self._get(id=job_id, params=params)
 
+    def build_for_rerun(self, job_id):
```

Comment:
Maybe this could be an internal method?
```suggestion
    def _build_for_rerun(self, job_id):
```

We could use this also for another `job_rerun()` method where the user can modify the tool parameters (and no output remapping).

Reply:
Or maybe a single `job_rerun()` method with a boolean variable `remap`?

Reply:
That could work, but that we would also need to add a `tool_inputs_update` dict parameter to let the user update tool parameters.

Reply:
I think I prefer to keep this as a normal (non-internal) method, isn't it possible that a user might want to use it directly?

Reply:
> I think I prefer to keep this as a normal (non-internal) method, isn't it possible that a user might want to use it directly?

What would be the use case?